jekyll/_cci2/gitlab-integration.adoc

```diff
@@ -20,39 +20,47 @@ This document walks you through integrating a GitLab project with CircleCI. The
 [#sign-up]
 == Sign up
 
-GitLab support is open to both new and existing users of CircleCI. Follow the steps on the xref:first-steps#gitlab-signup[Sign up and try CircleCI] page to:
+GitLab support is open to both new and existing users of CircleCI. Follow the steps on the xref:first-steps#gitlab-signup[Sign up and try CircleCI] page to connect your GitLab account to create your organization.
 
-* Connect your GitLab account to create your organization.
-* Create a project and connect a repository from your GitLab account.
+NOTE: You will need API access and write permissions on the repository you want to set up. Within GitLab, this is the “maintainer” role or higher.
 
-When you connect a repo with your CircleCI project, behind the scenes, CircleCI is registering a webhook within your GitLab repository. You may verify this once you have successfully created the project by navigating to your repository's **Settings > Webhooks** page.
+When you create a new organization and connect your GitLab account, you will be prompted to create a new project from a repository. You will also have the following options to set up a CircleCI configuration for your project:
 
-NOTE: You will need API access and write permissions on the repository you want to set up. Within GitLab, this is the “maintainer” role or higher.
+* **Fastest**: Use an existing `config.yml` in one of your repos.
++
+You can select any repository in your account that already has a `.circleci/config.yml`. The config file may be in the main branch, or a feature branch. You may select the same repo that you are setting up as a project, if you have already created a `.circleci` directory at the root of the repo, and added a `config.yml` file in that directory. You may also use _another_ repository that has previously been set up with a CircleCI configuration file. The `.circleci/config.yml` is imported into your new project--CircleCI automatically creates a new branch and adds a commit for you. 
```

Comment:
The options for fastest | faster | fast only show up if the repo you are trying to set-up has no circleci config file. So if you select the same repo here there won't be any config found.
Ideally, the user will want to select another repo with a config file in it.